server/middlewares/validateAuth.js

```diff
@@ -48,5 +50,66 @@ class validateAuth {
       });
     }
   }
+
+  /**
+     * This method handle the reset password request.
+     * @param {object} req The user's request.
+     * @param {object} res The response.
+     * @param {object} next pass to next method.
+     * @returns {object} The error and some error messages .
+     */
+  static validateResetPassword(req, res, next) {
+    const { error } = resetSchema.validate(req.body);
+    const valid = error == null;
+    if (valid) {
+      next();
+    } else {
+      const { details } = error;
+      const message = details.map(i => i.message).join(',');
+      res.status(422).json({
+        status: 422,
+        error: message
+      });
+    }
+  }
+
+  /**
+       * This method handle the update of reset password request.
```

Comment:
Update this comment as well in a correct and meaningful way.

Reply:
updated